engine/src/main/java/org/hibernate/search/spi/SearchIntegratorBuilder.java

```diff
@@ -423,10 +423,11 @@ else if ( metadataProvider.containsSearchMetadata( mappedClassIdentifier ) ) {
 
 		// Create all IndexManagers, configure and start them:
 		for ( XClass mappedXClass : rootIndexedEntities ) {
-			Class mappedClass = classMappings.get( mappedXClass );
+			final Class mappedClass = classMappings.get( mappedXClass );
```

Comment:
I think you could go a step further and make `classMappings` a `Map<XClass, IndexedTypeIdentifier>`. Not that it matters much for now, but that's probably where we're going, so...